src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/personalisation/legalrepresentative/LegalRepresentativeRemissionDecisionPartiallyApprovedPersonalisation.java

```diff
@@ -0,0 +1,62 @@
+package uk.gov.hmcts.reform.iacasenotificationsapi.domain.personalisation.legalrepresentative;
+
+import static java.util.Objects.requireNonNull;
+import static uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.AsylumCaseDefinition.*;
+import static uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.AsylumCaseDefinition.LEGAL_REPRESENTATIVE_EMAIL_ADDRESS;
+
+import com.google.common.collect.ImmutableMap;
+import java.util.Collections;
+import java.util.Map;
+import java.util.Set;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.AsylumCase;
+import uk.gov.hmcts.reform.iacasenotificationsapi.domain.personalisation.EmailNotificationPersonalisation;
+
+@Component
+public class LegalRepresentativeRemissionDecisionPartiallyApprovedPersonalisation implements EmailNotificationPersonalisation {
+
+    private final String partiallyApprovedTemplateId;
+    private final String iaExUiFrontendUrl;
+
+    public LegalRepresentativeRemissionDecisionPartiallyApprovedPersonalisation(
+        @Value("${govnotify.template.recordRemissionDecision.partiallyApproved.legalRep.email}") String partiallyApprovedTemplateId,
+        @Value("${iaExUiFrontendUrl}") String iaExUiFrontendUrl
+
```

Comment:
You need to inject the `CustomerServicesProvider customerServicesProvider` in order to have access to the customer services telephone and email address.

Reply:
Changed